src/backend/distributed/metadata/dependency.c

```diff
@@ -282,6 +288,26 @@ GetAllDependenciesForObject(const ObjectAddress *target)
 }
 
 
+/*
+ * GetAllCitusDependedDependenciesForObject returns all the dependencies
+ * which are owned by citus extension for the target.
+ */
+List *
+GetAllCitusDependedDependenciesForObject(const ObjectAddress *target)
```

Comment:
One idea I'm considering is to move `GetAllCitusDependedDependenciesForObject` under `src/test` as well. But, in that case, we'd need to include `src/test` in `src/backend` which sounds awkward.
Do you have any comments on this?

Reply:
I thought the same but we still need to expose 2 or 3 functions. I think the current way, in which we expose client functions in backend folder while we encapsulate internal stuff in test folder, sounds better.